### ISSUE TITLE
Extension completely freezes when opening a diff

### ISSUE THREAD

**Issue by tmsholy** [Saber2pr/vsc-file-git-history#9]
## Description

If I click on a file (`slurm.sh` in the screenshot) under the "Changes" section to open the diff, the extension just dies and shows endless loading. It does not update when switching between files anymore and I have to reload the window to fix it.

![Screenshot 2024-11-17 at 10 33 06](https://github.com/user-attachments/assets/abc03184-4b82-4904-ad83-17a5364a6e72)

## Platform

macOS 15.1

## Version

v0.0.25

**Comment by Saber2pr** [comment]
Yes, this issue pops up occasionally, and the reason isn’t clear. Reloading the window usually fixes it. I guess I’ll need to spend some time debugging it.

**Comment by Saber2pr** [comment]
Published at v0.0.28, the bug is fixed, maybe..
